tests/testthat/testFilter.R

```diff
@@ -0,0 +1,24 @@
+# testFilter.R
```

Comment:
- test that it works correctly on valid input
- test that it does not produce corrupt output from corrupt input